module/VuFindTheme/src/VuFindTheme/ResourceContainer.php

```diff
@@ -130,11 +130,102 @@ public function addCss($css)
     public function addJs($js)
     {
         if (!is_array($js) && !is_a($js, 'Traversable')) {
-            $js = [$js];
+            $this->addJsEntry($js);
+        } elseif (isset($js['file'])) {
```

Comment:
Is there any reason not to combine this condition with the above if clause, since both cases do the same thing?